extensions/raindrop-io/src/components/BookmarkForm.tsx

```diff
@@ -113,7 +113,12 @@ type BookmarkFormProps = {
 };
 
 async function getLinkTitle(link: string) {
-  return fetch(link)
+  let url = link.trim();
+  if (!url.match(/^https?:\/\//)) {
+    url = `https://${url}`;
+  }
```

Comment:
Duplicated code; should we move it to a new file and reference it in both files? Maybe we should also make it a bit more robust, or doesn’t that make sense?